Tests/pkacalc_test.py

```diff
@@ -4,6 +4,11 @@
 from subprocess import call
 import sys
 import unittest
+import logging
+
+
+ACCEPTABLE_ERROR = 0.001
```

Comment:
I would add the comment describing the rationale for the `ACCEPTABLE_ERROR` (from the comments on th PR).

Reply:
Addressed in 9cafb41e060c3b96c145492d95e8e5c60ea0b6fc